index.rst

```diff
@@ -769,9 +819,14 @@ Token Acceptance Guarantee
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The LSP API aspect services intend to guarantee all requests received that a given API service
```

Comment:
Perhaps replace with "The token proxy guarantees that the API service tokens that it issues to other Aspects will actually be usable when given to the API service."